### PR TITLE
Feature/swap ownership reject request

### PR DESCRIPTION
**Pull request by mksrivastava**

### CODE REVIEW

File: src/app/core/model/my-workplaces.model.ts [hunk 1]
```diff
@@ -36,3 +36,8 @@ export interface WorkPlaceReference {
   name: string;
   uid: string;
 }
+
+export enum RejectOptions {
+  NO = 'No, continue with no reason',
```
Comment (pjshumphreys) [inline src/app/core/model/my-workplaces.model.ts]: Does this refer to the ownership change request being denied? If so, a reason must be provided surely?

File: src/app/shared/components/reject-request-dialog/reject-request-dialog.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RejectRequestDialogComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```
Comment (pjshumphreys) [inline src/app/shared/components/reject-request-dialog/reject-request-dialog.component.spec.ts]: I think we should start having more unit tests than just the component could be instantiated. Don't add empty test suites like this.

Comment (mksrivastava) [reply src/app/shared/components/reject-request-dialog/reject-request-dialog.component.spec.ts]: this is the default one created automatically through command. So I'll remove it.

File: src/app/core/model/my-workplaces.model.ts (hunk 1 above)
Comment (mksrivastava) [reply src/app/core/model/my-workplaces.model.ts]: No this is the option which displays when reject request dialog open.

**Pull request merged**